custom-resource-handlers/src/pgp-secret.ts

```diff
@@ -0,0 +1,135 @@
+import aws = require('aws-sdk');
+import crypto = require('crypto');
+import fs = require('fs');
+import os = require('os');
+import path = require('path');
+import util = require('util');
+
+import cfn = require('./_cloud-formation');
+import _exec = require('./_exec');
+import lambda = require('./_lambda');
+import _rmrf = require('./_rmrf');
+import { resolveCurrentVersionId } from './_secrets-manager';
+
+const secretsManager = new aws.SecretsManager();
+const ssm = new aws.SSM();
+
+export async function main(event: cfn.Event, context: lambda.Context): Promise<void> {
+  try {
+    // tslint:disable-next-line:no-console
+    console.log(`Input event: ${JSON.stringify(event)}`);
+    const attributes = await handleEvent(event, context);
+    await cfn.sendResponse(event,
+                           cfn.Status.SUCCESS,
+                           attributes.SecretArn || event.PhysicalResourceId || context.logStreamName,
```

Comment:
Feels like if `SecretArn` is undefined/empty/null, it's a bug, isn't it?

Reply:
Successful DELETE has a blank SecretArn.